src/app/containers/ArticleTimestamp/index.jsx

```diff
@@ -27,6 +27,7 @@ const ArticleTimestamp = ({
 }) => {
   const {
     articleTimestampPrefix,
+    articleTimestampSuffix,
```

Comment:
Hey Fahad, the parent issue mentions string replacement as a possible solution moving forward. Have you considered this?

It would mean additional properties wouldn't need to be added to config files and components when a suffix is required over a prefix. It should also help simplify the way we think about this problem. 

[psammead-detokeniser](https://github.com/bbc/psammead/tree/latest/packages/utilities/psammead-detokeniser) can help us replace any token in a string with a value. It can be used in the following way:

```js
import detokenise from "@bbc/psammead-detokeniser"

const timestampTextTemplate = '%timestamp% मा अद्यावधिक';

// Replace %timestamp% in serviceTimestampText with the value of lastPublished.
const timestampText = detokenise('serviceTimestampText', { '%timestamp%': lastPublished })
```

Reply:
@simonsinclair the psammead-timestamp-container which does the text formatting already [has support for suffixes](https://github.com/bbc/psammead/blob/latest/packages/containers/psammead-timestamp-container/src/index.jsx#L44) out of the box. I feel a change to using the string replacement might conceal or render the already existing functionality obscure. Would like to know how others feel about this as well because the string replacement sounds cleaner although we already have something for the same purpose.

Reply:
Thanks, @rhenshaw56, I wasn't aware of that. It makes sense to me to move forward with the current approach.